src/pages/admin/reports/components/funds-expenditures-section/components/funds-expenditures-toolbar/FundsExpendituresToolbar.module.scss

```
@@ -175,3 +175,53 @@
+    z-index: 20;
+    padding: 6px 8px;
+}
+
+.menu-item {
+    background: transparent;
+    border: none;
+    padding: 8px 12px;
+    width: 100%;
+    text-align: left;
+    @include type.small-text-regular;
+    color: c.$blue-900;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)